recipe/meta.yaml

```diff
@@ -71,17 +68,20 @@ requirements:
     - setuptools
     - cython
     - {{ mpi }}  # [mpi != 'nompi']
-    - tempest-remap  # [tempest == 'tempest']
+    - tempest-remap 2.0.2  # [tempest == 'tempest']
```

Comment:
The TempestRemap API changed significantly between 2.0.2 and 2.0.3, and Moab 5.1.0 isn't compatible with the newer release.